packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/saml2assert/overview.md

```diff
@@ -1,8 +1,10 @@
 ## About the SAML 2.0 Assertion grant
 
-The SAML 2.0 Assertion flow is intended for a client app that wants to use an existing trust relationship without a direct user approval step at the authorization server. It enables a client application to obtain an authorization from a valid, signed SAML assertion from the SAML Identity Provider. The client app can then exchange it for an OAuth access token from the OAuth authorization server. For example, this flow is useful when you want to fetch data from APIs that only support delegated permissions without prompting the user for credentials.
```

Comment:
Why did you remove the last sentence?

Reply:
I move the full sentence into flow-diagram. The actual bearer types seemed liek to much detailf or an overview. On reflection it is rather missing the final step isn't it. Have added 

 The client then makes a request for an access token in exchange for the SAML 2.0 Assertion, and uses the access token to request resources from the resource server.